khmer/kfile.py

```diff
@@ -25,7 +25,7 @@ def check_input_files(file_path, force):
     """
     mode = None
 
-    if file_path == '-':
+    if file_path == '-' or file_path == '/dev/stdin':
```

Comment:
adding `or S_ISCHR(mode)` to line 45 (plus import on line 15) and not hardcoding the path here will be more effective and flexible.